docs/api/permissions.rst

```diff
@@ -0,0 +1,332 @@
+|                |                        | in the bucket.                   |
++----------------+------------------------+----------------------------------+
+| Collection     | ``write``              | Ability to write and read all    |
+|                |                        | objects in the collection.       |
+|                +------------------------+----------------------------------+
+|                | ``read``               | Ability to read all objects in   |
+|                |                        | the collection.                  |
+|                +------------------------+----------------------------------+
+|                | ``records:create``     | Ability to create new records    |
+|                |                        | in the collection.               |
++----------------+------------------------+----------------------------------+
+| Record         | ``write``              | Ability to write and read the    |
+|                |                        | record.                          |
+|                +------------------------+----------------------------------+
+|                | ``read``               | Ability to read the record.      |
+|                |                        |                                  |
++----------------+------------------------+----------------------------------+
+| Group          | ``write``              | Ability to write and read the    |
+|                |                        | group.                           |
+|                +------------------------+----------------------------------+
+|                | ``read``               | Ability to read the group.       |
+|                |                        |                                  |
++----------------+------------------------+----------------------------------+
+
+
+By default the ``write`` permission is given to the creator of an
+object.
+
+
+.. note::
+
+  There is no ``delete`` permission: Anyone with the ``write`` permission on an
+  object can also edit its associated permissions and delete it.
+
+
+Principals
+==========
+
+During the authentication phase, a set of :term:`principal`s for the current
+authenticated *user* will be bound to to the request.
+
+The main principal is considered the **user id**, and follows this formalism:
+``{type}:{identifier}`` (e.g. for Firefox Account: ``fxa:32aa95a474c984d41d395e2d0b614aa2``).
+
+There are two special principals:
+
+- ``system.Authenticated``: All users that are authenticated, no matter the
+  authentication mean.
+- ``system.Everyone``: Anyone (authenticated or anonymous). Using this
+  principal is useful when a rule should apply to all users.
+
+.. note::
+
+    A user can also be another application (in order to provide *service to
+    service* authentication).
+
+Get the current user id
+-----------------------
+
+The currently authenticated *user id* can be obtained on the root url.
```

Comment:
> [...] be obtained from the root URL.

URL is an acronym and should be capitalised. I would recommend checking the rest of the documentation to ensure consistency (just like for ID).